static/translate.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en" ng-app="mdrlTranslationApp">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="">
+
+    <title>SparkPost: SendGrid Template Translation</title>
+
+    <link href="vendor/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link href="vendor/font-awesome/css/font-awesome.min.css" rel="stylesheet">
+    <link href="main.css" rel="stylesheet">
+
+    <link rel="icon" href="favicon-16.png" sizes="16x16" />
+    <link rel="icon" href="favicon-32.png" sizes="32x32" />
+    <link rel="icon" href="favicon-48.png" sizes="48x48" />
+    <link rel="shortcut icon" href="favicon.ico" />
+
+    <script src="vendor/ace-builds/src-min-noconflict/ace.js"></script>
+    <script src="vendor/angular/angular.js"></script>
+    <script src="vendor/angular-sanitize/angular-sanitize.min.js"></script>
+    <script src="vendor/angular-ui-ace/ui-ace.js"></script>
+    <script src="vendor/jquery/dist/jquery.min.js"></script>
+    <script src="vendor/bootstrap/dist/js/bootstrap.min.js"></script>
+    <script src="vendor/angular-bootstrap/ui-bootstrap-tpls.js"></script>
+    <script src="translate.js"></script>
+  </head>
+  <body ng-controller="TranslatorControl">
+    <div class="titlebar">
+      <a href="https://www.sparkpost.com">
+        <img class="navbar-logo" src="https://developers.sparkpost.com/images/logo-sparkpost-white.png" alt="SparkPost" width="150">
+      </a>
+    </div>
+    <div ng-show="loading" style="position: absolute; width: 100px; height: 100px; top: calc(50% - 70px); left: calc(50% - 50px);"> LOADING
+    </div>
+    <div class="container-fluid">
+      <div class="jumbotron">
+        <div class="row">
+          <div class="col-md-8 col-md-offset-2">
+            <p>From here you can translate a template from SendGrid to SparkPost. To get started, paste your SendGrid template into the editor on the left below.</p>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-2 col-md-offset-5 text-center">
+            <button ng-disabled="loading" ng-click="translate()" class="btn btn-success btn-lg">Translate!</button>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-md-8 col-md-offset-2">
+          <uib-alert ng-repeat="alert in alerts" type="{{alert.type}}" close="closeAlert($index)">
+            <div><pre ng-show="alert.pre" ng-bind="alert.msg"></pre></div>
+            <div ng-show="!alert.pre" ng-bind="alert.msg"></div>
+          </uib-alert>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-sm-6">
+          <h3>SendGrid Template</h3>
+          <div id="sendgridEditor" ui-ace="{mode:'handlebars', onLoad: mdlEditorLoaded}"></div>
+
+          <label>Custom delimiter<br/>
```

Comment:
What can we do to clean up the delimiter field? Some ideas:

 - ask for just 1 delimiter
 - prefill with common delimiters from the docs (e.g. `-`, `%`)
 - separate out transactional and marketing campaign translation flows